veangle-guides/incentives.md

```diff
@@ -0,0 +1,16 @@
+---
+description: Earning ANGLE rewards
+---
+
+# Earning ANGLE rewards
+
+To earn ANGLE distributed as part of the weekly emissions, users can deposit sanTokens, open long positions on the protocol with Perps, or deposit LP (liquidity providers) tokens. 
+
+You can find all the available pools in the [Incentives page](https://dao.angle.money/#/stake) of the DAO App. 
+
+After staking your tokens, you will start accruing ANGLE. You can claim your tokens from each card, or from the `Claim Rewards` button on your left to claim all rewards from Perps and boosted gauges in one transaction. 
```

Comment:
Let's add a screenshot and say that you can decide not to claim from a particular gauge/perp if you feel that you have not accumulated enough rewards. The more gauges/things you'll claim the more expensive the transaction is going to be